RockLib.Messaging.CloudEvents/CloudEvent.cs

```diff
@@ -397,15 +441,25 @@ public virtual string ToJson(bool indent = false)
             };
 
             foreach (var attribute in Attributes)
+            {
                 jobject.Add(attribute.Key, JToken.FromObject(attribute.Value));
+            }
 
             if (_data is byte[] binaryData)
+            {
                 jobject["data_base64"] = Convert.ToBase64String(binaryData);
+            }
             else if (_data is string stringData)
             {
                 JToken dataToken;
-                try { dataToken = JToken.Parse(stringData); }
-                catch { dataToken = new JValue(stringData); }
+                try
+                {
+                    dataToken = JToken.Parse(stringData);
+                }
+                catch (Exception ex) when (ex is JsonReaderException)
```

Comment:
The `when` seems unnecessary.

```suggestion
                catch (JsonReaderException ex)
```